.mailmap

```diff
@@ -319,23 +348,30 @@ Justinas Lingys <jlingys@connect.ust.hk>
 Kacper Kowalik <xarthisius@gentoo.org>
 Kaleb S. Keithley <kkeithle@redhat.com>
 Kaleb S. Keithley <kkeithle@redhat.com> <kaleb@redhat.com>
+Kamoltat Sirivadhna <ksirivad@redhat.com> Kamoltat (Junior) Sirivadhna <ksirivad@redhat.com>
+Kamoltat Sirivadhna <ksirivad@redhat.com> Kamoltat <ksirivad@redhat.com>
 Kanika Murarka <kmurarka@redhat.com> <murarkakanika@gmail.com>
 Kapil Sharma <ksharma@suse.com>
 Karol Mroz <kmroz@suse.com> <kmroz@suse.de>
 Kefu Chai <kchai@redhat.com> <kefu@redhat.com>
 Kefu Chai <kchai@redhat.com> <tchaikov@gmail.com>
 Ken Dreyer <kdreyer@redhat.com> <ktdreyer@redhat.com>
+Kenneth Van Alstyne <Kenny.VanAlstyne@softiron.com> kvanals <Kenny.VanAlstyne@softiron.com>
 Kiefer Chang <kiefer.chang@suse.com>
 Kiseleva Alyona <akiselyova@mirantis.com>
 Kongming Wu <wu.kongming@h3c.com>
+Kotresh Hiremath Ravishankar <khiremat@redhat.com> Kotresh HR <khiremat@redhat.com>
 Kuanlong Li <likuanlong@sangfor.com.cn>
+Kyle McGough <kmcgough@digitalocean.com> Kyle <mcgoughboy@gmail.com>
 Kyr Shatskyy <kyrylo.shatskyy@suse.com> <kyrylo.shatskyy@suse.de>
 Kévin Caradant <kevin.caradant@gmail.com>
+lichaochao <lichaochao2_yewu@cmss.chinamobile.com> licc <lichaochao2_yewu@cmss.chinamobile.com>
 Lan De <lan.de3@zte.com.cn>
 Lan Liu <liulan@umcloud.com> <rjerk.whatever@gmail.com>
 Laszlo Boszormenyi <gcs@debian.hu>
-Laura Flores <lflores@redhat.com>
+Laura Flores <lflores@redhat.com> <ljflores@redhat.com>
```

Comment:
You can remove `<ljflores@redhat.com>`, as it is from a time I entered my email incorrectly.

Can you replace with my IBM email? `<lflores@ibm.com>`

Reply:
> You can remove <ljflores@redhat.com>, as it is from a time I entered my email incorrectly.

i added this during the https://tracker.ceph.com/projects/ceph/wiki/Ceph_contributors_list_maintenance_guide#Fixing-the-contributor-list step because you were showing up as a duplicate. this makes sure that even the typo'd commits get credited to you and your organization

> Can you replace with my IBM email? <lflores@ibm.com>

i added your ibm address and its affiliation. your commits under the red hat address will continue to be attributed to the red hat organization, but your individual stats will show up as `Laura Flores <lflores@ibm.com>`

Reply:
Thanks! Will switch it